src/modules/users/entities/role.entity.ts

```diff
@@ -0,0 +1,14 @@
+import { Column, Entity, OneToMany, PrimaryColumn } from 'typeorm';
+import { User } from './user.entity';
+
+@Entity()
+export class Role {
+  @PrimaryColumn()
+  role_id: number;
```

Comment:
we are using a CamelCase naming style, so the entity keys should be named in followed way
to name columns in database with underscore use option _name_ of _PrimaryColumn/Column_ decorators.

- [ ] rename to CamelCase